### PR TITLE
Milestone : Awesome books, a complete website with navigation

### PR DESCRIPTION
**Pull request by jenadiusnicholaus**
# In This milestone added:

1. A Navigation bar.
2. Three content sections:

      - Books list.
      - Add book form.
      - Contact info.
      
  Finally when you click on any of the links, the main area changes to display only one of the three content sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,30 +5,62 @@
...
-             <h2 class="form-heading">Add new book </h2> 
-            <input type="text" name="title" id="title" placeholder="Title" />
-            <br /><br />
-            <input type="text" name="author" id="author" placeholder="Author" />
-            <br /><br />
-            <div class="button-container"><button id="addbtn" type="button">Add</button>
-            </div>
-          </form>
         </div>
       </section>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly try to match the wireframe specification for the design of the books.

| Your design | Wireframe |
| --- | --- |
| image | image |

- [ ] Kindly remove `Add new book` on that page as it exists only on the `Add new` page.

File: index.html [hunk 2]
```diff
@@ -5,30 +5,62 @@
...
+         
+          <input type="text" name="title" id="title" placeholder="Title" />
+          <br /><br />
+          <input type="text" name="author" id="author" placeholder="Author" />
+          <br /><br />
+          <div class="button-container">
+            <button id="addbtn" type="button">Add</button>
+          </div>
+        </form>
+      </section>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly remove the `All Awesome books` text from the `Add new` page to match the wireframe specification.

| Your design | Wireframe |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -5,30 +5,62 @@
...
+          <p>
+            Do have any question on want to say : "Hello" ? <br />
+            You can reach out to us : <br />
+            .Our e-mail : example@gmail.com <br />.Our phone-number : 07848888888
+            <br />
+            Our address : Streename, 000,City Country
+          </p>
+        </div>
+       
+      </section>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly remove the `All Awesome books` and `Add new book` text from the `contact` page to match the wireframe specification.

| Your design | Wireframe |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -5,30 +5,62 @@
...
+      <footer></footer>
```
Comment (gemmen29) [inline index.html]: Kindly add the footer for all pages to match the wireframe specification as now there is no footer on all pages.
image

File: index.html [hunk 5]
```diff
@@ -5,30 +5,62 @@
...
+        <h2 class="awesome-books">Awesome Books</h2>
+        <nav>
+          <ul class="nav-links">
+            <li id ='booklist' class="link">List</li>
+            <li id ='add-book' class="link">Add new</li>
+            <li id="contactlink" class="link">Contact</li>
+          </ul>
+        </nav>
       </header>
       <section class="hero">
```
Comment (gemmen29) [inline index.html:30]: - [ ] [OPTIONAL] Kindly add the current date to match the wireframe specification and it's in the requirement of the project. You can use the luxon library.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job.  :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**